<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainvindow_gui_plugin</class>
 <widget class="QWidget" name="Mainvindow_gui_plugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2193</width>
    <height>1104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Induló Adatok</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Primál változók száma:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="termteny"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Korlátozó feltételek száma:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="korlfelt"/>
              </item>
              <item>
               <widget class="QPushButton" name="button_kezd">
                <property name="text">
                 <string>Induló értékek megadása</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="button_kanonikus">
                <property name="text">
                 <string>Átalakított alak 
generálása</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_indulo_matrix">
                <property name="text">
                 <string>Induló mátrix
generálása</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="button_szamol">
                <property name="text">
                 <string>Számol
</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="text">
                 <string>Egy Lépést számol</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>Végigszámol</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Induló alak</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget2">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Átalakított alak</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_2"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget3">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Induló tábla</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_3"/>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget4">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Előző lépés után</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tableView_4"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget5">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Utolsó lépés után</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tableView_5"/>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
